addons/purchase_stock/models/stock.py

```diff
@@ -18,6 +18,7 @@ class StockMove(models.Model):
         'Purchase Order Line', ondelete='set null', index=True, readonly=True)
     created_purchase_line_id = fields.Many2one('purchase.order.line',
         'Created Purchase Order Line', ondelete='set null', readonly=True, copy=False)
+    forecasted_issue = fields.Boolean(related='purchase_line_id.forecasted_issue')
```

Comment:
Well, IMO, it shouldn't be only for move related to purchase order line, but for all incoming (maybe internal) transfer.
Maybe use the `forecast_availability` and modify the `forecast_widget` to take in account incoming transfers.
IMO, the issue with that, is there will be 2 forecasted buttons with complete different logic/fields: one for outgoing (delivery - already present with custom widget + 2 fields `forecast_availability` and `forecast_expected_date`) and a other (your) related to `forecasted_issue` for incoming (only related to a purchase, and this is wrong). What do you think ?

Reply:
Yes it makes total sense to 'unifiy' the forecasted buttons, I'll have a look at it.